lte/gateway/c/core/oai/tasks/nas/esm/esm_ebr.c

```diff
@@ -486,21 +488,25 @@ status_code_e esm_ebr_set_status(
   OAILOG_FUNC_IN(LOG_NAS_ESM);
 
   if (emm_context == NULL) {
-    OAILOG_ERROR(LOG_NAS_ESM, "ESM-FSM   - emm context null... \n");
+    OAILOG_ERROR_UE(
```

Comment:
emm_context does not exist in this case, let's not change this

Reply:
Resolved